samples/app/src/index.js

```diff
@@ -8,31 +10,21 @@ import './index.css';
 // it's up to you to detect the locale of your user
 // (by url, cookie, header X-Accept-Language, etc)
 const matches = document.location.search.match(/lang=(.*)/);
-const locale = (Array.isArray(matches) && matches.pop()) || 'en';
+let locale = (Array.isArray(matches) && matches.pop()) || 'en';
+
+const availableLocales = ['fr', 'en'];
+if (!availableLocales.includes(locale)) {
+  locale = 'en';
+}
```

Comment:
This show the usage of @gandi/react-translate's `negociateLocale` (which takes a list of possible user locales by order of precedence (fallbacks last, overrides first)), and show another possible source of user locale as a fallback (the browser's accept-language HTTP header).